modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -364,6 +374,45 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private void _hasPreventDeletionType(long objectDefinitionId) {
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipLocalService.getObjectRelationships(
+				objectDefinitionId);
+
+		_hasPreventTypeRelationship = false;
```

Comment:
I was thinking, and I believe that when we declare a boolean variable without initializing, it will always be initialized as false because the variable is a primitive type. So I believe we could delete this line, what do you think?

Reply:
I make a short research about it and is right. I will erase this line